specification/maps/data-plane/Microsoft.Maps/Render/preview/2.0/render.json

```diff
@@ -304,83 +211,39 @@
             }
           },
           {
-            "$ref": "#/parameters/Language"
+            "$ref": "../../../Search/preview/1.0/search.json#/parameters/Language"
           },
           {
-            "$ref": "#/parameters/View"
+            "$ref": "../../../Search/preview/1.0/search.json#/parameters/View"
           }
         ],
         "produces": [
-          "application/json"
+          "application/json",
+          "image/jpeg",
+          "image/png",
+          "image/pbf",
+          "application/vnd.mapbox-vector-tile"
         ],
         "responses": {
           "200": {
-            "description": "OK",
+            "description": "The tile returned from a successful API call.",
             "schema": {
-              "$ref": "#/definitions/MapTileResultv2"
+              "type": "object",
+              "format": "file",
+              "readOnly": true
+            },
+            "headers": {
+              "Content-Type": {
+                "type": "string",
+                "description": "The content-type for the response."
+              }
             }
           },
-          "400": {
-            "$ref": "#/responses/400"
-          },
-          "401": {
-            "$ref": "#/responses/401"
-          },
-          "403": {
-            "$ref": "#/responses/403"
-          },
-          "404": {
-            "$ref": "#/responses/404"
-          },
-          "500": {
-            "$ref": "#/responses/500"
+          "default": {
+            "$ref": "../../../Common/preview/1.0/common.json#/responses/default"
           }
         }
       }
     }
-  },
-  "definitions": {
-    "ODataErrorResponse": {
-      "type": "object",
-      "description": "This response object is returned when an error occurs in the Maps API.",
-      "properties": {
-        "error": {
-          "$ref": "#/definitions/ODataError"
-        }
-      }
-    },
-    "ODataError": {
-      "type": "object",
-      "description": "This object is returned when an error occurs in the Maps API.",
-      "properties": {
-        "code": {
-          "type": "string",
-          "readOnly": true,
-          "description": "The ODataError code."
-        },
-        "message": {
-          "type": "string",
-          "readOnly": true,
-          "description": "If available, a human readable description of the error."
-        },
-        "details": {
-          "type": "array",
-          "items": {
-            "$ref": "#/definitions/ODataError"
-          }
-        },
-        "target": {
-          "type": "string",
-          "readOnly": true,
-          "description": "If available, the target causing the error."
-        }
-      }
-    },
-    "MapTileResultv2": {
-      "description": "The tile returned from a successful API call.",
-      "type": "string",
-      "format": "binary",
-      "readOnly": true
-    }
```

Comment:
This definition is still being referenced above. Should it be kept?

Reply:
You mean MapTileResultv2? It's not referenced anymore since 200 response definition was changed to describe a file as a response.

Reply:
I see. I thought the response still referenced this as a schema. My mistake.